infra/modules/curi/jupyter_sdk/dns.tf

```diff
@@ -0,0 +1,8 @@
+resource "aws_route53_record" "jupyter_cname" {
+  allow_overwrite = true
+  zone_id         = var.hosted_zone_id
+  name            = "jupyter-sdk.${var.hosted_zone}"
+  type            = "CNAME"
+  ttl             = 3600
```

Comment:
any reason you chose an hour? is that the minimum?

Reply:
the ttl on this dns record shouldn't matter because it's pointing to the cloudfront distribution which shouldn't change, the value of the redirect is controlled by the s3 bucket.